app/assets/styles/partials/components/_character-limit.scss

```diff
@@ -0,0 +1,20 @@
+textarea.js-charlimit-exceeded {
```

Comment:
This styling should be part of the `input` block as`js-` classes are hooks only. This would be more consistent with the rest of the codebase and also allow styling to be applied without JS. Something like `input__limit`.